src/router/index.js

```diff
@@ -101,12 +110,12 @@ class Router {
   // 参考: https://router.vuejs.org/ja/guide/essentials/navigation.html
   push(path) {
     history.pushState(null, null, path);
-
     this.emit(path);
   }
 
-  replace() {
-
+  replace(path) {
+    history.replaceState(null, null, path);
```

Comment:
```suggestion
    history.replaceState(history.state, null, path);
```

これにしてもらえるとstateを引き継げるのでroutefulの整合性取れます

Reply:
@phi-jp  あ、routeful使ってないですかね？どっちにしろ後でそうなると思うので使ってなくても上のコメントのとおりにしておいてもらえると

Reply:
ok